main/src/main/java/cgeo/geocaching/ui/dialog/SimpleDialog.java

```diff
@@ -510,7 +510,7 @@ private <T> void selectItems(final ItemSelectModel<T> options, final Consumer<Se
         if (model.selectSetSupplier != null) {
             this.setNeutralButton(model.selectSetActionText);
         }
-        if (!selectionConfirmedViaButton) {
+        if (!selectionConfirmedViaButton && buttonClickAction == null) {
```

Comment:
I used the negative button for `Previous filter`, but to get it work, that both, negative and neutral buttons are shown (but no positive button), I had To get it work, I had to check for `buttonClickAction` in SimpleDialog.

Reply:
I guess that's ok because such a case is pretty rare.
In fact does it really make sense to have a dialog with negative button but no positive one?

Reply:
I used the negative button as a second neutral one, not as the real intention of positive / negative meaingn